### DISCUSSION TITLE
Tuya devices don't support Zigbee binding ?

### DISCUSSION THREAD

**Discussion by sigma415** [Koenkk/zigbee2mqtt#22949] [Devices]
Hello,

I tried various bindings (using Z2M frontend) with various Tuya device (see below) and could not manage one single functional binding.
What is your experience with binding of Tuya devices ? 
Are they in general not binding capable ?

I tried following Tuya devices, purchased via Aliexpress:

- 809WZT: Motion sensor
- TS0202_2: Motion sensor with scene switch (Linkoze LKMSZ001)
- TS0044: Wireless switch with 4 buttons
- TS011F_plug_1: Smart plug (with power monitoring)
- TS0001_power: Switch with power monitoring
- TS0001_switch_module: 1 gang switch module (Moes ZM-104-M)

Some of them don’t show the required clusters in the binding frontend, some don’t report the needed clusters and for some rare combinations binding seemed to be successful, but direct switching doesn’t work. :(

I tried binding now with 2 IKEA devices  (E2134 VALLHORN wireless motion sensor with LED2109G6 TRADFRI bulb E27, color/white spectrum) 
...... and binding of this combination works without any problems.  :)

**Comment by Koenkk** [comment]
Most of them dont (since they use a custom protocol)

**Comment by gatredev** [comment]
Hey @sigma415, were you able to figure out how to directly bind the device list you shared, or did you find another way to achieve this? Thanks!

**Comment by sigma415** [comment]
No, tuya's are cheap but doesn't support binding.

**Comment by gatredev** [comment]
Did you end up finding any other switches that support this, or did you decide to give up on direct binding?
